cmd/microcloud/main_init_preseed.go

```diff
@@ -564,55 +564,42 @@ func (p *Preseed) Parse(s *service.Handler, c *initConfig) (map[string]InitSyste
 			c.systems[peer] = system
 		}
 
-		// Check the preseed underlay network configuration against the available ips.
+		// TODO: call `s.Services[types.MicroOVN].(*service.OVNService).SupportsFeature(context.Background(), "custom_encapsulation_ip")`
```

Comment:
I suspect for the next MicroCloud LTS release we would anyway not recommend installing an OVN which isn't using microcluster `v2`?

But nevertheless good to have it.

Reply:
It's still not fully clear which tracks are going to support what (except for the LTS) so I think keeping these extension checks consistent throughout the code will be useful if we ever have a parallel non-long-term support track with some more loose restrictions.

Reply:
Worth removing this comment now and doing the actual check after https://github.com/canonical/microcloud/pull/398 got merged?

Reply:
Fixed.